w_common/lib/invalidation_mixin.dart

```diff
@@ -35,19 +35,19 @@ abstract class InvalidationMixin {
   /// and [ValidationStatus.cancelled] if invalidation is cancelled.
   Future<ValidationStatus> invalidate() {
```

Comment:
I think we should have a null check at the start of this method to ensure `_onValidate` is not `null`.

Reply:
Nevermind, the `invalid` check handles this case.